CharacterCreator/CharacterCreator/Models/Characters.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharacterCreator.Models
+{
+    public enum Sex
+    {
+        Male,
+        Female
+    }
+
+    public enum Alignment
+    {
+        LawfulGood,
+        NeutralGood,
+        ChaoticGood,
+        LawfulNeutral,
+        TrueNeutral,
+        ChaoticNeutral,
+        LawfulEvil,
+        NeutralEvil,
+        ChaoticEvil
+    }
+    public class Characters
```

Comment:
Character needs to also include `ClassId `and `RaceId `attributes (these are your foreign keys). The Class and Race references you have are references to the table, but not a specific value. Refer to image for setting up foreign key values:

<img width="218" alt="ex" src="https://user-images.githubusercontent.com/21206742/66729541-a9ac3200-ee11-11e9-90c9-7fc7fbf2f245.png">

Reply:
Fixed